Wireframe/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+    <header>
+      <h1>Git Made Easy</h1>
+      <h2>An explanation of the mysteries of Git.</h2>
+    </header>
+
+    <div class="main">
+      <article class="hero">
+        <img>
+        <h2>What is Git?</h2>
+        <p>Git is the backbone of modern software development, a powerful version control system that helps developers track changes, collaborate, and maintain project history. Whether you're working solo or in a large team, Git keeps your code organized and recoverable. Want to know how it works and why it's so essential? Click to dive in!
+        </p>
+        <a target="_blank" href="https://www.freecodecamp.org/news/what-is-git-and-how-to-use-it-c341b049ae61/"><button>READ MORE</button></a>
```

Comment:
Can a link contain a button? How would you test to find out?

Your website is automatically deployed as a preview on this PR, here : https://deploy-preview-120--cyf-ufd.netlify.app/wireframe/

What does the [validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-120--cyf-ufd.netlify.app%2Fwireframe%2F) tell you about this page?